ci/build-project.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+UNREAL_PATH=${1}
```

Comment:
Quote to avoid parameter expansion with spaces.

Let me show you pre-commit.com sometime, and how to wire it with shellcheck and shfmt - that should take no more than a half hour, and will automate most of the review I'm giving here.